railties/lib/rails/generators/rails/app/templates/github/ci.yml.tt

```diff
@@ -136,6 +136,7 @@ jobs:
 
       - name: Run tests
         env:
+          CI: true
```

Comment:
https://docs.github.com/en/actions/reference/workflows-and-actions/variables
I believe GitHub Actions does this by default